App/CrisisCleanup/Sources/Feature/Cases/CasesViewModel.swift

```diff
@@ -5,6 +5,8 @@ class CasesViewModel: ObservableObject {
     private let incidentSelector: IncidentSelector
```

Comment:
The cases screen does not show the profile picture in any way. This entire file can reset.